explore/src/main/scala/explore/constraints/ConstraintsPanel.scala

```diff
@@ -171,120 +174,122 @@ object ConstraintsPanel {
-              id = "ertype",
-              label = Label("Elevation Range", HelpIcon("constraints/main/er.md")),
-              value = state
-                .zoom(State.rangeType)
-                .withOnMod(updateElevationRange),
-              clazz = ExploreStyles.ElevationRangePicker
             ),
-            ReactFragment(
-              FormInputEV(
-                id = "minam",
-                label = "Min",
+            <.div(
+              ExploreStyles.FlexContainer,
+              ExploreStyles.ConstraintsElevationRangeGroup,
+              EnumViewSelect(
+                id = "ertype",
+                label = Label("Elevation Range", HelpIcon("constraints/main/er.md")),
                 value = state
-                  .zoom(State.airMass)
-                  .zoom(AirMassRange.min)
-                  .zoomSplitEpi(
-                    TruncatedRefinedBigDecimal.unsafeRefinedBigDecimal[AirMassRange.Value, 1]
-                  )
-                  .withOnMod(min => erView.set(state.get.airMass.copy(min = min.value))),
-                errorClazz = ExploreStyles.InputErrorTooltip,
-                errorPointing = LabelPointing.Below,
-                validFormat = ValidFormatInput
-                  .forRefinedTruncatedBigDecimal[AirMassRange.Value, 1](airMassErrorMsg)
-                  .composeValidFormat(
-                    ValidFormatNec.lte(
-                      TruncatedRefinedBigDecimal[AirMassRange.Value, 1](state.get.airMass.max).get,
-                      "Must be <= Max"
-                    )
-                  ),
-                changeAuditor = ChangeAuditor.accept.decimal(1),
-                clazz = ExploreStyles.ElevationRangeEntry
+                  .zoom(State.rangeType)
+                  .withOnMod(updateElevationRange),
+                clazz = ExploreStyles.ElevationRangePicker
               ),
-              FormInputEV(
-                id = "maxam",
-                label = "Max",
-                value = state
-                  .zoom(State.airMass)
-                  .zoom(AirMassRange.max)
-                  .zoomSplitEpi(
-                    TruncatedRefinedBigDecimal.unsafeRefinedBigDecimal[AirMassRange.Value, 1]
-                  )
-                  .withOnMod(max => erView.set(state.get.airMass.copy(max = max.value))),
-                errorClazz = ExploreStyles.InputErrorTooltip,
-                errorPointing = LabelPointing.Below,
-                validFormat = ValidFormatInput
-                  .forRefinedTruncatedBigDecimal[AirMassRange.Value, 1](airMassErrorMsg)
-                  .composeValidFormat(
-                    ValidFormatNec.gte(
-                      TruncatedRefinedBigDecimal[AirMassRange.Value, 1](state.get.airMass.min).get,
-                      "Must be >= Min"
+              ReactFragment(
+                FormInputEV(
+                  id = "minam",
+                  label = Label("Min", HelpIcon("constraints/main/minam.md")),
```

Comment:
Aren't these the buttons that Andy just removed?

Reply:
This is rebased on top of Andy's Pr and I tried to keep those changes but I'll review again